openfisca_us/variables/usda/snap.py

```diff
@@ -175,3 +175,16 @@ class is_disabled_or_elderly_for_snap(Variable):
     entity = Person
     definition_period = YEAR
     documentation = "Indicates that a person is defined as disabled or elderly based on the USDA definition"
+
+
+class is_homeless_for_snap(Variable):
```

Comment:
Let's make this a household-level `is_homeless` variable in https://github.com/PolicyEngine/openfisca-us/blob/master/openfisca_us/variables/demographic/household.py

We're a ways out from needing to model different homelessness definitions across agencies.

This variable doesn't need a formula.